eahub/config/settings.py

```diff
@@ -276,7 +278,16 @@ class DjangoEnv(Enum):
 
 WEBPACK_DEV_URL = env("WEBPACK_DEV_URL", default="http://localhost:8090/assets")
 
-SETTINGS_EXPORT = ["WEBPACK_DEV_URL", "DEBUG", "DJANGO_ENV", "ALGOLIA"]
+DOMAIN = env.str("DOMAIN", "localhost:8000")
+
+SETTINGS_EXPORT = [
+    "WEBPACK_DEV_URL",
+    "DEBUG",
+    "DJANGO_ENV",
+    "ALGOLIA",
+    "RECAPTCHA_PUBLIC_KEY",
```

Comment:
Why are we adding RECAPTCHA and DOMAIN here?

Reply:
they're needed in few templates. In short it's perfectly ok to add to this list everything, except the secrets as `RECAPTCHA_SECRET_KEY` which must never be rendered through HTML